sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -1894,4 +1894,3 @@ object QueryExecutionErrors {
     new RuntimeException("Unable to convert timestamp of Orc to data type 'timestamp_ntz'")
   }
 }
-
```

Comment:
@wangyum preserve the end line?

Reply:
No, we should not keep an empty line! We do keep \n at the end of the file, but not an empty line!